components/04-views/product-view/ProductView.html

```diff
@@ -0,0 +1,211 @@
+<alpaca-container>
+  <div class="product-view">
+    <alpaca-breadcrumbs
+      :breadcrumbs="breadcrumbs"
+    />
+    <div class="product-view__wrapper">
+      <div class="product-view__gallery">
+        <alpaca-gallery
+          :images="gallery.images"
+        />
+      </div>
+      <div class="product-view__main-details">
+        <div class="product-view__heading">
+          <alpaca-heading
+            tag="h1"
+            level="1"
```

Comment:
do we need tag and level, Is is not the same result?

Reply:
Should be, I found a bug in heading component validation, fixed